packages/i18n/src/create-i18n.js

```diff
@@ -395,7 +395,7 @@ export const createI18n = ( initialData, initialDomain, hooks ) => {
 			 */
 			result = /** @type { boolean } */ (
 				/** @type {*} */ hooks.applyFilters(
-					'i18n.hasTranslation',
+					'i18n.has_translation',
 					result,
 					single,
 					context,
```

Comment:
"domain" here is still being passed to filters as "default" rather than "undefined" for calls with no domain due to the default value in the method signature. That's different behaviour from the other `i18n.` filters so would be nice to standardise.

Reply:
Thanks for the reminder, fixed in 43c0fff 🚀